T2CC_APP/andriod app/COSC612StudentAndriod/app/src/main/res/layout-land/activity_main.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/username"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <TextView
+        android:id="@+id/Title"
+        android:textStyle="bold"
+        android:layout_width="390dp"
+        android:layout_height="34dp"
+        android:layout_marginBottom="32dp"
+        android:text="Teacher-To-Class Communication  "
```

Comment:
I think we should start using T2CC instead of the full name? It looks abit awkward with the full name